src/css/skins/beelden.less

```diff
@@ -42,6 +42,9 @@
         box-shadow: inset 0px 7px 1px -5px rgba(128,128,128,1);
         border-radius: .3em;
     }
+    &.jw-flag-audio-player .jw-controlbar {
+        bottom: 0; // overwrite the inset-controlbar mixin
+    }
```

Comment:
With the goal of reducing duplication, while also keeping the code together, the best solution seems to be modifying the mixin.